x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/editor/math_completion.ts

```diff
@@ -276,8 +276,13 @@ function getArgumentSuggestions(
       ) {
         possibleOperationNames.push(
           ...a.operations
-            .filter((o) =>
-              operation.requiredReferences.some((requirement) => requirement.input.includes(o.type))
+            .filter(
+              (o) =>
+                operation.requiredReferences.some((requirement) =>
+                  requirement.input.includes(o.type)
+                ) &&
+                o.operationType !== 'formula' &&
+                o.operationType !== 'math'
```

Comment:
We already have `o.hidden` as a filterable parameter here, please use that instead @dej611